pkgs/by-name/sp/speechd/package.nix

```diff
@@ -143,9 +143,15 @@ stdenv.mkDerivation (finalAttrs: {
 
   meta = {
     description =
-      "Common interface to speech synthesis" + lib.optionalString libsOnly " - client libraries only";
+      "Common high-level interface to speech synthesis"
+      + lib.optionalString libsOnly " - client libraries only";
     homepage = "https://devel.freebsoft.org/speechd";
-    license = lib.licenses.gpl2Plus;
+    changelog = "https://github.com/brailcom/speechd/blob/${finalAttrs.src.rev}/NEWS";
+    license = [
+      lib.licenses.gpl2Only
+      lib.licenses.gpl3Only
+      lib.licenses.lgpl21Only
+    ];
```

Comment:
This does not look right. I see no GPL-3.0 licensed code and according to https://github.com/brailcom/speechd#licensing, the daemon is mainly `GPL-2.0-or-later` and the libraries `LGPL-2.1-or-later`. dotconf is dynamically linked so it should not have an effect on the license of the daemon, IIRC.